packages/core/test/new-api/call.ts

```diff
@@ -15,7 +15,7 @@ describe("call", () => {
       return { contract1 };
     });
 
-    const constructor = new ModuleConstructor(0);
+    const constructor = new ModuleConstructor(0, []);
```

Comment:
I have no problem with duplication in tests, I think you have to justify DRY in tests due to the cognitive burden of abstractions.

However, I am wondering if we are over the line and a helper should be added to support our usage of construct here?

What do people think? @alcuadrado @zoeyTM

Reply:
I'm normally ok with some duplication in tests, so no strong preference here.

Reply:
also no strong preference from me; i'm fine either way.